includes/admin/views/builder/question.php

```diff
@@ -3,7 +3,7 @@
  * Builder question view
  *
  * @since   3.16.0
- * @version 3.27.0
+ * @version [version]
```

Comment:
I would like to see a changelog here, can you?

Reply:
done

Reply:
I think I've committed the suggested changes and got confused about this. Now fixed.